### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by shaqdeff**
- Created a to-do list structure with HTML, CSS and JavaScript.
- Used Webpack to set up the project.
- Followed all project and general requirements. 

Kindly review and provide feedback.

Regards.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  list.append(li);
+};
+
+// Function to create new todo object
+const addTodo = (description) => {
+  const todo = {
+    description,
+    completed: false,
+    index: Date.now(),
```
Comment (sja-thedude) [inline src/index.js]: - [ ] As per the project requirements stated below, you can see it says that the index should be a number. In your code, you have given the date as the value for the index. kindly fix it according to the requirements.

Screenshot 2022-06-06 at 3 40 14 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @shaqdeff, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters are passing
✔️ Professional Readme
✔️ Good PR Title and Description

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (shaqdeff) [reply src/index.js]: Hey @sja-thedude, 
Thanks for the review. Just changed that.

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @shaqdeff, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**